tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -144,6 +145,7 @@ public async Task VerifySpans(IImmutableList<MockSpan> spans, VerifySettings set
             settings.AddRegexScrubber(AppSecWafDurationWithBindings, "_dd.appsec.waf.duration_ext: 0.0");
             settings.AddRegexScrubber(AppSecRaspWafDuration, "_dd.appsec.rasp.duration: 0.0");
             settings.AddRegexScrubber(AppSecRaspWafDurationWithBindings, "_dd.appsec.rasp.duration_ext: 0.0");
+            settings.AddRegexScrubber(AppSecSpanIdRegex, "\"span_id\": XXX");
```

Comment:
Shouldn't you be checking that the span_id in the JSON blob references the parent spanid? Have you tried _not_ adding this scrubber, and seeing if you see `Id_3` etc instead?

Reply:
Yes, I tried, but it does not show Id_3 but the actual id instead.

Reply:
May be worth investigating why verify doesn't replace the real id with one of it's fake ones, could be a good fix to make at some point.